src/symbols.js

```diff
@@ -492,6 +492,7 @@ defineSymbol(math, main, rel, "\u2190", "\\gets");
 defineSymbol(math, main, rel, ">", "\\gt");
 defineSymbol(math, main, rel, "\u2208", "\\in");
 defineSymbol(math, main, rel, "\u2209", "\\notin");
+defineSymbol(math, main, rel, "\u0338", "\\not");
```

Comment:
So `\not` is a symbol *and* a function? Somewhat confusing, I think. Better add a comment there, indicating that this line isn't all there is to that command.

Why do you use the combining version of the glyph, instead of the plain `/`? After all, you're not combining with anything, and I'm not sure whether placing a combining character without a base character is sufficiently well specified to ensure consistent rendering across systems.